src/com/oltpbenchmark/benchmarks/tpcc/TPCCLoader.java

```diff
@@ -210,6 +214,33 @@ protected void EnableForeignKeyConstraints(Connection conn) {
     }
   }
 
+  protected void CreatePgSqlProcedures(Connection conn) throws SQLException {
+    try {
+      Statement st = conn.createStatement();
+
+      StringBuilder argsSb = new StringBuilder();
+      StringBuilder updateStatements = new StringBuilder();
+
+      argsSb.append("wid int");
+      for (int i = 1; i <= 15; ++i) {
+        argsSb.append(String.format(", i%d int, q%d int, y%d int, r%d int", i, i, i, i));
+        updateStatements.append(String.format(
+          "UPDATE STOCK SET S_QUANTITY = q%d, S_YTD = y%d, S_ORDER_CNT = S_ORDER_CNT + 1, " +
+          "S_REMOTE_CNT = r%d WHERE S_W_ID = wid AND S_I_ID = i%d;",
+          i, i, i, i));
+        String updateStmt =
+          String.format("CREATE PROCEDURE updatestock%d (%s) AS '%s' LANGUAGE sql;",
```

Comment:
nit: capitalize `sql`->`SQL`, also I think you can put the `LANGUAGE` clause before `AS` as it's clearer when reading procedures.

Reply:
Done.